skunkbooth/utils/helpers.py

```diff
@@ -0,0 +1,47 @@
+import logging
+from datetime import datetime
+from multiprocessing.queues import Queue
+from typing import List, Tuple
+
+from asciimatics.event import Event, KeyboardEvent
+from asciimatics.exceptions import StopApplication
+from asciimatics.screen import Screen
+
+from skunkbooth.data.defaults import PIC_DIR
+
+
+def global_shortcuts(event: Event) -> None:
+    """Event handler for global shortcuts"""
+    ctrlQCode = Screen.ctrl("q")
+    ctrlWCode = Screen.ctrl("w")
+    if isinstance(event, KeyboardEvent):
+        c = event.key_code
+        # Stop on q, esc, ctrl+q and ctrl+w
+        if c in (Screen.KEY_ESCAPE, ord("q"), ctrlQCode, ctrlWCode):
+            raise StopApplication("User pressed quit")
+
+
+def toggleFlag(flag: List[int]) -> None:
+    """Temp function for toggling video recording from inside screen"""
+    vidBuf = Queue(32767)
```

Comment:
vidBuf is a shared pipe between processes that needs to contain messages between processes, in this case a video filename. If you declare a new buffer in here, the value won't get sent out to the shared buffer outside this function. You can add function parameters if they're needed.

Reply:
got it. I will do the testing and will let you know.

Reply:
What happened with the testing ?